Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -2090,6 +2220,35 @@ public void Register(string sessionName, ProductUserId userIdToRegister)
             sessionInterface.RegisterPlayers(ref registerOptions, sessionName, OnRegisterCompleteCallback);
         }
 
+        /// <summary>
+        /// Registers possibly multiple users to the Session.
+        /// Users will be able to query the identities of Registered members in a Session,
+        /// and Registering is required for EOS Game Services to manage Maximum Users configurations.
+        /// When called by the Owner of a Session, the Registered user is reflected with EOS Game Services.
+        /// Other users should be informed by the Owner to Register users, so they have their information.
+        /// P2P messages can be used to inform the Owner and users when to run this method.
+        /// </summary>
+        /// <param name="sessionName">The name of the local Session to register the user to.</param>
+        /// <param name="userIdsToRegister">
+        /// The ProductUserId of the users to Register to the Session.
+        /// If this array is empty, the function exits without performing an operation.
+        /// Uses <paramref name="sessionName"/> for ClientData to identify the Session to register the user to.
+        /// </param>
+        public void Register(string sessionName, ProductUserId[] userIdsToRegister)
```

Comment:
This is a great circumstance in which to utilize the `params` keyword. Opinions may differ on best practices, but if you're unfamiliar with the keyword this would be a good circumstance to become familiar with it (regardless of whether this is an instance in which to make use of it).

Reply:
It feels like this is best called as an array. `params` seems more fit for something reading in command line arguments or something.

Reply:
Not sure how to use this limited set of emoji to suggest "I contest this change idea"... But ah, I don't think this one should be params-y, or at least there should also be this array option. Unless params covers the array option, which I think maybe it does..?